suite-native/device-manager/src/components/DeviceItemContent.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { Icon, IconName } from '@suite-common/icons';
+import { HStack, Box, Text } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import {
+    selectDeviceNameById,
+    DeviceRootState,
+    PORTFOLIO_TRACKER_DEVICE_ID,
+    selectDeviceLabelById,
+} from '@suite-common/wallet-core';
+import { TrezorDevice } from '@suite-common/suite-types';
+import { TypographyStyle } from '@trezor/theme';
+import { useActiveColorScheme } from '@suite-native/theme';
+
+type Props = {
+    deviceId: TrezorDevice['id'];
+    isPortfolioLabelDisplayed?: boolean;
+    deviceNameTextVariant?: TypographyStyle;
+};
+
+export const DeviceItemContent = ({
+    deviceId,
+    isPortfolioLabelDisplayed = true,
+    deviceNameTextVariant = 'body',
+}: Props) => {
+    const activeColorScheme = useActiveColorScheme();
+
+    const deviceName = useSelector((state: DeviceRootState) =>
+        selectDeviceNameById(state, deviceId),
+    );
+    const deviceLabel = useSelector((state: DeviceRootState) =>
+        selectDeviceLabelById(state, deviceId),
+    );
+
+    const isPortfolioTrackerDevice = deviceId === PORTFOLIO_TRACKER_DEVICE_ID;
+
+    // TODO: when we enable remember mode, icon representing disconnected device have to be handled.
+    const trezorDeviceIcon: IconName =
```

Comment:
Can you again make some exhaustive check here?

Reply:
I've checked with designer, and there will be the same Icon for all the possible trezor models. So IMO there is no need for exhaustive check here.